### ISSUE TITLE
Skin Lesion Images classification

### ISSUE THREAD

**Issue by aaradhyasinghgaur** [TAHIR0110/ThereForYou#147]
**Is your feature request related to a problem? Please describe.**
To classify between differnt skin lesions . The dataset for ISIC 2019 contains 25,331 images available for the classification of dermoscopic images among nine different diagnostic categories:

  -  Melanoma
  -  Melanocytic nevus
  -  Basal cell carcinoma
  -  Actinic keratosis
  -  Benign keratosis (solar lentigo / seborrheic keratosis / lichen planus-like keratosis)
  -  Dermatofibroma
  -  Vascular lesion
  -  Squamous cell carcinoma
  -  None of the above`
  
  dataset I'll use :- https://www.kaggle.com/datasets/salviohexia/isic-2019-skin-lesion-images-for-classification

**Describe the solution you'd like**

  1.  Data Preparation: Collect and preprocess a balanced dataset of real and fake face images, including normalization, resizing, and augmentation.
  2.  Base Model Selection: EfficientNetB0,VGG16 , Xception , InceptionV3 like 5 different models excluding its top layers, to leverage its learned features.
  3.  Model Construction: Add custom layers on top of the base model for binary classification, compiling with appropriate loss and metrics.
  4.  Initial Training: Train the model with the base layers frozen to only update the new layers.
  5.  Fine-Tuning: Unfreeze some or all of the base model layers and continue training with a lower learning rate to fine-tune the entire network.
    6.) EDA analysis.
    7.) Comaprioson using performance matrices such as accuracy scores , confusion matrix etc.

**Describe alternatives you've considered**

**Additional context**

@TAHIR0110 kindly assign me this issue.

**Comment by TAHIR0110** [comment]
@kyra-09 assigned!
